doc/en/how-to/writing_plugins.rst

```diff
@@ -151,23 +151,28 @@ that ``pytest`` finds your plugin module.  Entry points are
 a feature that is provided by :std:doc:`setuptools:index`. pytest looks up
 the ``pytest11`` entrypoint to discover its
 plugins and you can thus make your plugin available by defining
-it in your setuptools-invocation:
+it in your ``pyproject.toml`` file.
 
-.. sourcecode:: python
+.. sourcecode:: toml
+
+    # sample ./pyproject.toml file
+    [build-system]
+    requires = ["setuptools"]
+    build-backend = "setuptools.build_meta"
```

Comment:
In https://docs.pytest.org/en/latest/explanation/goodpractices.html we are using `hatchling`, we probably should follow suit here:

```suggestion
    [build-system]
    requires = ["hatchling"]
    build-backend = "hatchling.build"
```